docs/pages/x/api/data-grid/selectors.json

```
@@ -13,6 +13,20 @@
     "description": "Get the aggregation model, containing the aggregation function of each column.\nIf a column is not in the model, it is not aggregated.",
     "supportsApiRef": true
   },
+  {
+    "name": "gridColumnDefinitionsSelector",
+    "returnType": "GridStateColDef<any, any, any>[]",
+    "category": "Columns",
+    "description": "Get the columns as an array.",
+    "supportsApiRef": true
+  },
+  {
+    "name": "gridColumnFieldsSelector",
+    "returnType": "string[]",
+    "category": "Columns",
+    "description": "Get the field of each column.",
```

We can probably provide more information.

```suggestion
    "description": "Get an array containing the field of each column in the order rendered on screen.",
```